Documentation/network/servicemesh/external-ingress-policy.rst

```diff
@@ -0,0 +1,71 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_external_ingress_policy:
+
+*************************
+External Lock-down Policy
+*************************
+
+The previous example doesn't include any network policy, so all the external traffic is allowed by default.
+Let's apply network policy to lock down the external traffic.
```

Comment:
Again, I think it would make sense here to refer to the policy as an object, similar to how we do here: https://docs.cilium.io/en/stable/network/kubernetes/policy/#ciliumclusterwidenetworkpolicy

```suggestion
Let's apply a `CiliumNetworkPolicy` to lock down external traffic.
```